app/src/main/res/values/dimens.xml

```diff
@@ -1,5 +1,7 @@
 <resources>
     <!-- Default screen margins, per the Android Design guidelines. -->
-     <dimen name="activity_horizontal_margin">16dp</dimen>
-     <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="activity_horizontal_margin">16dp</dimen>
+    <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="size_text_activities">25sp</dimen>
```

Comment:
A couple of notes on `dimens`:

1) It's common practice to use dimens (in `sp`, `dp`, etc.) that are multiples of 4. In this case this `25sp` should be `24sp` (or `16sp`, `20sp`, `28sp`, `32sp`, etc.). 

The reason for this is that there is a difference that is significant enough to the naked eye between each size by separating each dimen by 4. This helps you to avoid having lot's of different dimens at the end of the day. 

If a designer asks for a text to be in... let's say, 31 sp? Well then you approximate that requirement to `32sp`.

Another reason for using multiples of 4 (and therefore divisible by 4) is that usually dimens can be converted to fractions more easily. Take the example of a header that occupies `24dp` of width. You could then add in another line an icon on the left of `6dp` (1/4), a subtext of `12dp` (1/2) and another icon to the right of `6dp` (1/4) and it would be exactly the same width as the header. That would be much harder to do with a dimen of `25dp', as it would not be divisible by 4.

2) Always make groups of dimens based on their role. For example:

```xml
<!--  Font Sizes -->
<dimen name="font_size_regular">16sp</dimen>
<dimen name="font_size_big">24sp</dimen>
<dimen name="font_size_huge">72sp</dimen>

<!-- Spacings -->
<dimen name="spacing_small">4dp</dimen>
<dimen name="spacing_medium">8dp</dimen>
<dimen name="spacing_big">16dp</dimen>
```